device/cluster.h

```diff
@@ -278,8 +278,8 @@ class tt_device {
     virtual void configure_tlb(
         chip_id_t logical_device_id,
         tt_xy_pair core,
-        std::int32_t tlb_index,
```

Comment:
Do we generally want to work towards removing std prefix everywhere? Asking in general, if that is what we want to follow I am ok with that. I see a lot of inconsistent usage of this

Reply:
Yes, in the spirit of aligning with tt-metal's style let us favor integer types without the prefix.

Reply:
Makes sense